lab-anthony/lib/greet.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+function greet(name){
+  console.log('Hello ' + process.argv[2]);
```

Comment:
You should move this console.log statement into the greet-cli.js file to make your function conform exactly to the assignment specifications.